### ISSUE TITLE
WAITING: Recreate uranus_occs_earthbased metadata index files when able

### ISSUE THREAD

**Issue by rfrenchseti** [SETI/rms-data-projects#79]
Once @esimpsons3ti updates the PDS4 index file generator to support XPath columns, etc., we need to regenerate the metadata files for the uranus_occs_earthbased bundleset to their final permanent version. At the same time we have to update OPUS to handle the new index file format.

**Comment by matthewtiscareno** [comment]
What is the status of this?  Why is it marked "Waiting"?  It is linked to #234?

**Comment by rfrenchseti** [comment]
Yes, we can consider this a duplicate of #234 and close this one.
